docs/DEPLOYMENT_GUIDE.md

```diff
@@ -0,0 +1,136 @@
+# 🚀 Deployment Guide for Starknet Smart Contracts by Stark Cairo Nodes
+
+This guide provides step-by-step instructions for deploying smart contracts on Starknet using **Remix IDE**. It covers the installation, compilation, and deployment processes for proper use of the IDE.
+
+---
+
+## 📚 Table of Contents
+1. [⚙️ Prerequisites](#prerequisites)
+2. [🔌 Installing the Starknet Plugin](#installing-the-starknet-plugin)
+3. [🛠️ Compiling Smart Contracts](#compiling-smart-contracts)
+4. [🚢 Deploying Contracts](#deploying-contracts)
+5. [📄 Example Deployment](#example-deployment)
+6. [🔗 Troubleshooting Tips](#troubleshooting-tips)
+
+---
+
+## ⚙️ 1. Prerequisites
+Ensure you have the following before starting:
+- **Remix IDE**: [Access Remix IDE](https://remix.ethereum.org/)
+- Starknet Wallet (Argent X or Braavos).
+- Access to the `stark-cairo-nodes` repository: [GitHub Repo](https://github.com/KaizeNodeLabs/stark-cairo-nodes).
+
+## 🔌 2. Installing the Starknet Plugin
+Follow these steps to install the `Starknet` plugin on Remix IDE:
+
+1. Open **Remix IDE**.
+2. Navigate to the **Plugin Manager**.
+3. Search for **Starknet** and click "Activate".
+4. Click to **Install** the plugin.
+
+Once activated, you will see a Starknet tab in Remix.
+
+## 🛠️ 3. Compiling Smart Contracts
+To compile contracts in Remix using the Starknet plugin:
+
+1. Open the **Starknet** tab.
+2. In the File Explorer, navigate to the `contracts/helloWorld/hello_world.cairo` file from the repository.
```

Comment:
Is this directory available in remix? I think we need first to create a new workspace and then create a new file, after that copy and paste the code from the repository